src/modules/auth/components/AuthModal/AuthModal.tsx

```diff
@@ -25,6 +25,12 @@ const AuthModal: FC<AuthModalProps> = (props) => {
 		await AuthService.redirectToLoginHetArchief(queryParams, router);
 	};
 
+	const onRegisterHetArchief = async () => {
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const { showAuth, ...queryParams } = query;
+		await AuthService.redirectToRegisterHetArchief(queryParams, router);
+	};
+
 	const onLoginMeemoo = async () => {
```

Comment:
Consistentie in naamgeving, `...HetArchief` vs `...Meemoo`?

Reply:
there are 2 identity providers: 
* hetarchief 
* meemoo

so there are also 2 login methods:
* onLoginHetArchief
* onLoginMeemoo

New visitors can only register in the hetarchief identity provider (idp), so there is only one register handler:
* onRegisterHetArchief

I think that is consistent. Do you agree?